richardekong007/Property-pro-lite-api
# #167003462 change approach to saving image URL

**What does the PR do?**
- Merges updates to develop branch.
- Updates involve adding multer and using it as the multipart middleware to support the extraction and storage of secured image URL from cloudinary.

**Description of task**
- Add multer to package.json to handle multipart requests
- Modify api.v1.js by using multer as a multipart middleware in POST/property route
- Modify index.property.specs.js while re-testing POST/property route

**Manual test**
- clone this repository and change directory to the local project root
- install all dependencies by executing: **npm install**
- start the express server by issuing the command:**npm run start**, and wait for the prompt *Listening on port - 3999* to be displayed
- head to postman and initiate a post request on route: **api/v1/property**, then check the **form data** radio button and create the following entries shown below, and hit the send the button.

![image](https://user-images.githubusercontent.com/19517166/60512110-a611f680-9ccb-11e9-9745-1056c69198f8.png)


[start #167003462]
